.github/workflows/docs-ci.yml

```diff
@@ -4,9 +4,45 @@ on:
   pull_request:
     branches: [ main ]
   push:
-    branches: [ 'main', 'release/v*' ]
+    branches: [ '**', 'release/v*' ]
```

Comment:
I'm not sure I would universally change this to `**`. The convention we use in other workflows is to have a specific prefix. I think if you did `branches: [ 'main', 'docs/**', 'release/v*' ]`, that would be more in line w/ our current conventions (see `build-ci.yml`).

Reply:
That was for testing. Fixed.